frontend/src/pages/LandingPage.js

```diff
@@ -1,33 +1,36 @@
 // SPDX-License-Identifier: MIT
 // SPDX-FileCopyrightText: 2021 Christoph Ehm <christoph.ehmendoerfer@campus.tu-berlin.de>
+// SPDX-FileCopyrightText: 2021 Dominic Heil <d.heil@campus.tu-berlin.de>
 
-import * as React from "react";
+import React from "react";
 
 import WelcomeBanner from "../components/Banner";
 import HeaderTypography from "../components/HeaderTypography";
 
 import Grid from "@mui/material/Grid";
 
 import AnnouncementPage from "./AnnouncementPage";
+import ExampleNftImg22 from "../assets/img/nftExamples/image_part_022.png";
+import ExampleNftImg21 from "../assets/img/nftExamples/image_part_021.png";
+import NftCardHorizontallyScrollableList from "../components/NftCardHorizontallyScrollableList";
+let dummyData = [
+	{ title: "Nürnberg NFT 021", price: "1.0", nftPageUrl: "/drop", imgUrl: ExampleNftImg21, description: "There is something cool about this text. When this text gets to long the text will be automatically cut off. My absolutely biggest secret is that I love cookies." },
+	{ title: "Nürnberg NFT 022", price: "0.0001", nftPageUrl: "/drop", imgUrl: ExampleNftImg21, description: "text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text " },
+	{ title: "Nürnberg NFT 023", price: "0.0002", nftPageUrl: "/drop", imgUrl: ExampleNftImg22, description: "This text can be descriptive." },
+];
```

Comment:
It would be nice to have this variable in it's own imported JS file.

Reply:
It will be deleted anyways. So no need to overthink this.